README.md

```diff
@@ -6,10 +6,10 @@ This repository contains documentation maintained at https://doc.dovecot.org.
 Prerequisites
 -------------
 
-Initialize your python virtualenv environment
```

Comment:
Just for readability, could you split out the README.md changes to a different commit. They do not really have anything to do with Submission.

Thank you for your work!